integration/backendscheduler/backendscheduler_test.go

```diff
@@ -64,26 +64,183 @@ func TestBackendScheduler(t *testing.T) {
 	t.Logf("Endpoint: %s", e)
 
 	scheduler := util.NewTempoTarget("backend-scheduler", configFile)
-	worker := util.NewTempoTarget("backend-worker", configFile)
-	require.NoError(t, s.StartAndWaitReady(scheduler, worker))
+	require.NoError(t, s.StartAndWaitReady(scheduler))
 
 	// // Setup tempodb with local backend
 	// tempodbWriter := setupBackend(t, tmpDir)
 	tempodbWriter := setupBackendWithEndpoint(t, &cfg.StorageConfig.Trace, e)
 
-	// // Push some data to a few tenants
-	for i := 0; i < tenantCount; i++ {
-		testTenant := tenant + strconv.Itoa(i)
-		populateBackend(ctx, t, tempodbWriter, testTenant)
+	time.Sleep(5 * time.Second)
```

Comment:
any reasons to add sleep over `StartAndWaitReady` for all services we want to wait on?


it's okay but I would like us to avoid using sleep when we can avoid.